public/bitemp_graph.js

```diff
@@ -471,12 +474,20 @@ var barChart = function() {
         })
         .text(function(d) {
           var str = '';
-          setDefaultDispPropBehavior(d);
-          if (displayProperty.indexOf('.') === -1) {
-            str = d.content[displayProperty];
+          if(window.location.href === 'http://localhost:3000/search') { 
+            str = d.content.uri;
           }
           else {
-            str = path(d, 'content.' + displayProperty);
+            setDefaultDispPropBehavior(d);
+            if (displayProperty.indexOf('.') === -1) {
+              str = d.content[displayProperty];
+              if(str instanceof Array) {
```

Comment:
Sorry I sent you this but I realize that this is unnecessary since you already handle that case in setDeafaultDispPropBehavior()